action/base.go

```diff
@@ -78,6 +79,9 @@ func (t *SignedTx) SignedBytes() []byte {
 }
 
 func ValidateBasic(data []byte, signerAddr []Address, signatures []Signature) error {
+	if len(signatures) < len(signerAddr) {
```

Comment:
Should exactly match, right? Shall we replace < with !=  ?

Reply:
Not exactly as u can have a required signers (as 2 for example), but u could put 3 signatures for check

Reply:
The signers and signatures are supposed to be in order, that is how it gets verified.

Reply:
I can remove, but if signers will be mismatched u will get a panic at this line:
pkey := signatures[i].Signer

if you don't like this solution, then propose another one. But anyway, if I have a 1 signer and we got a multiple signatures, how It be resolved?